themes/cakephp/static/cake.css

```diff
@@ -32,6 +32,8 @@
     font-style: normal;
 }
 
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,400italic);
```

Comment:
@bcrowe @import must be move to the first statement (before the other @font-face) for me to work locally.
It's part of the [specification](http://www.w3.org/TR/CSS2/cascade.html#at-import)

Appart of this, it looks great :clap: